File: src/Penghasilan.java
import java.util.Scanner;

public class Penghasilan {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int penghasilan = scanner.nextInt();
        double pajak = 0.0;

        if (penghasilan >= 0 && penghasilan <= 50) {
            pajak = 0.05;
        } else if (penghasilan >= 51 && penghasilan <= 250) {
            pajak = 0.15;
        } else if (penghasilan >= 251 && penghasilan <= 500) {
            pajak = 0.25;
        } else if (penghasilan > 500) {
            pajak = 0.30;
        }

        int potonganPajak = (int) (penghasilan * pajak * 1000000);
        int penghasilanBersih = (penghasilan * 1000000) - potonganPajak;

        System.out.println("Penghasilan kotor = " + penghasilan + " juta Rupiah");
        System.out.println("Pajak " + (int) (pajak * 100) + "% = " + potonganPajak);
        System.out.println("Penghasilan bersih = " + penghasilanBersih);
    }
}

File: src/com/company/lms/Penghasilan.kt
package com.company.lms

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)

    val pajak: Long
    val besarPajak: Long
    val penghasilanBersih: Long

    var penghasilanKotor = scanner.nextLong()

    pajak = when (penghasilanKotor) {
        in 0..50 -> 5
        in 51..250 -> 15
        in 251..500 -> 25
        else -> 30
    }

    println("Penghasilan Kotor = $penghasilanKotor Juta Rupiah")

    penghasilanKotor *= 1000000
    besarPajak = (penghasilanKotor * pajak) / 100
    penghasilanBersih = penghasilanKotor - besarPajak

    println("Pajak ${penghasilanKotor - besarPajak}% = Rp.$besarPajak")
    println("Penghasilan bersih Rp.$penghasilanBersih")
}